app/views/alchemy/admin/pages/_pages.html.erb

```diff
@@ -0,0 +1,117 @@
+<%- tree = pages_tree_from(root) -%>
+<%- tree.each_with_index do |(page, level), i| -%>
+  <%- page_has_children = tree[i + 1] && tree[i + 1][1] > level -%>
+  <li id="page_<%= page.id %>" class="page_level_<%= "#{level} #{page.page_layout}" %>" data-slug="<%= page.slug %>" data-restricted="<%= page.restricted? %>" data-visible="<%= page.visible? %>" data-external="<%= page.redirects_to_external? %>">
+    <div class="sitemap_page<%= page.locked ? ' locked' : '' %>" name="<%= page.name %>">
+      <div class="sitemap_left_images">
+        <% sitemap_folder_link(page) unless level == 1 || !page_has_children || @sorting %>
```

Comment:
```erb
<%= sitemap_folder_link(page), ..
```

without loud erb the folder link isn't visible, but maybe it's intended by you, because the folding doesn't work anymore, because of missing `page` partial